support/cas-server-support-consent-webflow/src/main/java/org/apereo/cas/web/consent/CasConsentReviewController.java

```diff
@@ -0,0 +1,144 @@
+package org.apereo.cas.web.consent;
+
+import org.apereo.cas.consent.ConsentDecision;
+import org.apereo.cas.consent.ConsentEngine;
+import org.apereo.cas.consent.ConsentRepository;
+import org.apereo.cas.web.support.WebUtils;
+import org.apereo.inspektr.common.spi.PrincipalResolver;
+import org.pac4j.core.config.Config;
+import org.pac4j.core.context.J2EContext;
+import org.pac4j.core.engine.CallbackLogic;
+import org.pac4j.core.http.J2ENopHttpActionAdapter;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.stereotype.Controller;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.ResponseBody;
+
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import javax.servlet.http.HttpSession;
+import java.util.Collection;
+import java.util.HashMap;
+import java.util.HashSet;
+import java.util.Map;
+
+/**
+ * This is {@link CasConsentReviewController}.
+ *
+ * @author Arnold Bergner
+ * @since 5.2.0
+ */
+@Controller("casConsentReviewController")
+@RequestMapping("/consent")
+public class CasConsentReviewController {
+    private static final Logger LOGGER = LoggerFactory.getLogger(CasConsentReviewController.class);
+
+    private final Config pac4jConfig;
+    private final String defaultUrl;
+    
+    /**
+     * The consent repository.
+     */
+    private final ConsentRepository consentRepository;
+    
+    /**
+     * The consent engine.
+     */
+    private final ConsentEngine consentEngine;
+
+    public CasConsentReviewController(final ConsentRepository consentRepository,
+            final ConsentEngine consentEngine, final Config pac4jConfig, final String defaultUrl) {
+        this.consentRepository = consentRepository;
+        this.consentEngine = consentEngine;
+        this.pac4jConfig = pac4jConfig;
+        this.defaultUrl = defaultUrl;
+    }
+    
+    /**
+     * Show consent decisions.
+     *
+     * @param request  the request
+     * @param response the response
+     * @return the view where json data will be rendered
+     */
+    @GetMapping
+    public String showConsent(final HttpServletRequest request,
+                                        final HttpServletResponse response) {
+        return "casConsentReviewView";
```

Comment:
You may want to turn this string into a constant and use it throughout the class.